composer.json

```diff
@@ -9,7 +9,7 @@
     "php": ">=5.3.0"
   },
   "require-dev": {
-    "phpunit/phpunit": "~3.7.0"
+    "phpunit/phpunit": "^9.5"
```

Comment:
That's some upgrade! Only 6 majors and the app still works, doesn't it? ;-)

Reply:
...all green for me 👍

Reply:
There were two small changes in naming but test passed :)